FEATURES.md

```diff
@@ -0,0 +1,17 @@
+# Proposing Features
+
+A feature request or proposal is an issue opened on a repo in opensearch-project that states an end-user problem, and proposes a solution in the form of a feature. It should meet the following criteria.
```

Comment:
I find it helpful to have an example to help frame expectations around process recommendations like this.  Is there an existing feature request that fits this mold or maybe could be updated to align well with the intentions of this doc?

Reply:
If you know of a really good one I'd be happy to add it!

Reply:
I like the idea of adding guidance and while an example would be useful if we can always update later.  I'll keep watching.